ocs_ci/framework/conf/default_config.yaml

```diff
@@ -123,6 +123,7 @@ UPGRADE:
   ocp_channel: "candidate-4.5"
   ocp_upgrade_path: "quay.io/openshift-release-dev/ocp-release"
   ocp_arch: "x86_64"
+  upgrade_logging_channel: "4.4"
```

Comment:
I guess this "4.4" here is cause of OCP 4.5 is not GAed yet, so please can you open W/A issue to change this back once 4.5 is GAed? And add link to this issue to comment above this line?

Reply:
so can we track with this issue itself, If @petr-balogh would suggest I would add a point in the same issue https://github.com/red-hat-storage/ocs-ci/issues/2256 or you would want me track this with new issue ?

Reply:
I am for open new issue with the title like:   `Fix issues after OCP 4.5 GA` and we can add it there as one of the bullet point.   I actually opened issue here:  https://github.com/red-hat-storage/ocs-ci/issues/2390  so you can add the link to this issue in the comment one line above this, so if someone will take a look in this config they will know the reason why in 4.5 config we have 4.4.